### PR TITLE
Feat/655 send backend logs opensearch

### PR DESCRIPTION
**Pull request by RMCampos**
# Description

Send SPAR Backend API (only this, for learning and testing, later Oracle-api as well) to OpenSearch.

Fixes [FSADT2-655](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-655)

**Note**: If you're going to try this change locally, you need to create a directory at the root directory, called `logs` and give all users write permissions. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

This was partially tested locally, but we need to test with deployments to make sure everything works.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

## Further comments

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-202-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-202-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-202-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

**Comment by RMCampos** [comment]
Officially open for review now 🎉 

@craigyu @DerekRoberts @paulushcgcj

**Review by DerekRoberts — APPROVED** [review]
That was a lot of work! Great job. :)

File: backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -13,8 +13,11 @@ spring.security.oauth2.resourceserver.jwt.jwk-set-uri = ${keycloak-auth}/protoco
 management.endpoint.health.show-details = always
 
 # Others
-nr-spar-backend-version = ${NR_SPAR_BACKEND_VERSION:#{'snapshot'}}
-server.allowed.cors.origins = ${ALLOWED_ORIGINS:#{'http://127.*, http://localhost:300*'}}
+nr-spar-backend-version = ${NR_SPAR_BACKEND_VERSION:#{'development'}}
+nr-spar-team-email-address = FSA.Delivery.Team2@gov.bc.ca
+nr-spar-ecs-version = 8.8
+nr-spar-backend-env-opensearch = ${NR_SPAR_BACKEND_ENV_OPENSEARCH:development}
+server.allowed.cors.origins = ${ALLOWED_ORIGINS:#{'http://localhost:8090'}}
```
Comment (craigyu) [inline backend/src/main/resources/application.properties:20]: Just a reminder, you mentioned something about removing one of the `development` in the meeting

**Review by craigyu — APPROVED** [review]
Really nice work Ricardo!

File: backend/src/main/resources/application.properties (hunk 1 above)
Comment (RMCampos) [reply backend/src/main/resources/application.properties:20]: Thanks for that!

**Pull request merged**